starfish/pipeline/features/spots/detector/__init__.py

```diff
@@ -35,7 +33,7 @@ def add_to_parser(cls, subparsers):
         cls.spot_finder_group = spot_finder_group
 
     @classmethod
-    def _cli(cls, args: argparse.Namespace, print_help=False):
+    def _cli(cls, args, print_help=False):
```

Comment:
why this change?

Reply:
Makes pycharm super upset.

Reply:
If I could disable inspections on that parameter in the IDE, would that be a good middle ground?

Reply:
That's cool, just drop the type hint then.

It's frustrating that the type hinting system isn't better. :(

Reply:
Agree. :(